modules/sdk-core/src/bitgo/wallet/iWallet.ts

```diff
@@ -93,6 +93,7 @@ export interface PrebuildTransactionOptions {
   enableTokens?: TokenEnablement[];
   nonce?: string;
   preview?: boolean;
+  isTss?: boolean;
```

Comment:
Can you explain how this is looking to be used?

Reply:
We are looking to use this flag when building eth TSS transactions through prebuildTransaction. Eventually this is passed into prebuildTxWithIntent and populateIntent so that we can have a properly formed intent when calling wallet-platform.